src/FSLibrary/MissionSorobanConfigUpgrades.fs

```diff
@@ -107,5 +107,17 @@ let sorobanConfigUpgrades (context: MissionContext) =
                       txMaxWriteBytes = Some(50000 * 2) }
                 (System.DateTime.UtcNow.AddSeconds(20.0))
 
-            formation.RunLoadgen coreSet context.GenerateSorobanUploadLoad
+            let peer = formation.NetworkCfg.GetPeer coreSet 0
+            let startingOpCount = peer.GetMetrics().LedgerOperationApply.Value.Count
+
+            // There is a race condition where previously valid TXs in the TX queue may become invalid
+            // once the tx size limit is reduced. Allow 5% of failures to account for this race.
+            try
+                formation.RunLoadgen coreSet { context.GenerateSorobanUploadLoad with skiplowfeetxs = true }
```

Comment:
this shouldn't throw because loadgen with skiplowfeetxs=true is expected to pass